openstack/identity/v3/tokens/testing/requests_test.go

```diff
@@ -248,6 +248,31 @@ func TestCreateProjectNameAndDomainIDScope(t *testing.T) {
 }
 
 func TestCreateProjectNameAndDomainNameScope(t *testing.T) {
+	options := tokens.AuthOptions{UserID: "fenris", Password: "g0t0h311"}
+	scope := &tokens.Scope{System: true}
+	authTokenPost(t, options, scope, `
+		{
+			"auth": {
+				"identity": {
+					"methods": ["password"],
+					"password": {
+						"user": {
+							"id": "fenris",
+							"password": "g0t0h311"
+						}
+					}
+				},
+				"scope": {
+					"system": {
+						"all": true
+					}
+				}
+			}
+		}
+	`)
+}
+
+func TestCreateSystemScope(t *testing.T) {
```

Comment:
@zhsj I think you want the body of the above test in this function?

Reply:
I copied the `TestCreateProjectNameAndDomainNameScope` to `TestCreateSystemScope`, but I ended up modifying the `TestCreateProjectNameAndDomainNameScope` func...

Now fixed.